hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -63,6 +63,10 @@
 
   public HiveSyncTool(HiveSyncConfig cfg, HiveConf configuration, FileSystem fs) {
     super(configuration.getAllProperties(), fs);
+    //Set Hive meta store URI which enables synchronize to remote hive
+    if (cfg.hiveMetaStoreUri != null) {
+      configuration.set("hive.metastore.uris",cfg.hiveMetaStoreUri);
```

Comment:
might be a naive question. Can you help me understand where else can I find the configs where we translate hoodie params to hive params. I tried to search in our repo, but couldn't find any. or is this the first one we do like this? for eg, wrt JDBC url, hoodie param is named as "hoodie.datasource.hive_sync.jdbcurl", but we directly pass this value while creating a connection to hive. But are there other configs which gets converted from hoodie param to hive param somewhere? Basically trying to understand why this new config is handled specially here and not along w/ any other such translations.

Reply:
This may be a deviation of my understanding, I will turn off this pr

Reply:
In HoodieSparkSqlWriter.scala,

private def buildSyncConfig(basePath: Path, parameters: Map[String, String]): HiveSyncConfig = {
    val hiveSyncConfig: HiveSyncConfig = new HiveSyncConfig()
    hiveSyncConfig.basePath = basePath.toString
    hiveSyncConfig.baseFileFormat = parameters(HIVE_BASE_FILE_FORMAT_OPT_KEY)
    hiveSyncConfig.usePreApacheInputFormat =
      parameters.get(HIVE_USE_PRE_APACHE_INPUT_FORMAT_OPT_KEY).exists(r => r.toBoolean)
    hiveSyncConfig.databaseName = parameters(HIVE_DATABASE_OPT_KEY)
    hiveSyncConfig.tableName = parameters(HIVE_TABLE_OPT_KEY)
    hiveSyncConfig.hiveUser = parameters(HIVE_USER_OPT_KEY)
    hiveSyncConfig.hivePass = parameters(HIVE_PASS_OPT_KEY)
...


@nsivabalan  - This converts from the hudi params to the hive config.